src/applications/static-pages/dependency-verification/components/dependencyVerificationHeader.jsx

```diff
@@ -1,26 +1,13 @@
 import React from 'react';
-import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
 
 const DependencyVerificationHeader = () => {
   return (
     <header>
-      <h1 className="vads-u-font-size--h2">Dependents on your VA benefits</h1>
```

Comment:
I think this h1 is still needed, or else Axe may throw an error. At least, it did for me when I was testing this earlier 🤷‍♂️

Reply:
Oh wait, just noticed you passed a title to the modal itself. Ignore me 😄